sv_inline.h

```diff
@@ -1003,6 +1003,90 @@ Perl_sv_setpv_freshbuf(pTHX_ SV *const sv)
     return SvPVX(sv);
 }
 
+/* Perl_int2str_table: lookup table containing string representations
```

Comment:
With the "E" flag this and the function should only be defined when PERL_EXT or PERL_CORE is defined.